build/workflow/Makefile

```diff
@@ -81,6 +83,29 @@ ${TESTS}: setup-testing
 >@${MESSAGE} end "Passed." "Failed."
 >@rm -f status_code
 
+${STANDALONE_MODULE_TEST}: ${MODULE_TESTS}
+>@test_target=$$(echo $@ | sed 's/..\/..\/${TEST_LOCATION}\/${MODULE_NAME}\///g') ; \
+    test_script=$${test_target/test-/} ; \
+    test_name=$${test_script/module_tests\/no_docker_test_/} ; \
+    test_name=$${test_name/unit_tests\/no_docker_test_/} ; \
+    extension=$$(echo $$test_name | grep -o "\.sh\|\.py$$" ) ; \
+    test_name=$${test_name/$$extension/} ; \
+    test_name=$$(echo "$$test_name" | tr '_' ' ' ) ; \
+    ${MESSAGE} start "\u2503 $$test_name"
+>@test_target=$$(echo $@ | sed 's/..\/..\/${TEST_LOCATION}\/${MODULE_NAME}\///g') ; \
+    test_script=$${test_target/test-/} ; \
+    test_name=$${test_script/module_tests\/no_docker_test_/} ; \
+    filename="module_tests/$$(basename $$test_script)" \
+    extension=$${filename##*.} ; \
+    executor=none-cmd ; \
+    if [[ "$$extension" == "py" ]]; then executor=python; fi ; \
+    if [[ "$$extension" == "sh" ]]; then executor=bash; fi ; \
+    cd ../../${TEST_LOCATION}/${MODULE_NAME}/ ; \
+    $$executor $$filename ${DOCKER_ORG_NAME}-development/${DOCKER_REPO_PREFIX}-development:latest ; \
+    echo "$$?" > status_code
```

Comment:
This line probably isn't running, hence no status_code, hence the error you observed.

Reply:
I can confirm that it runs since it produces Nextflow file output.

Reply:
No, the error you reported is proof that this specific line did not run.

```sh
cat: status_code: No such file or directory
```

Reply:
There is also a predecessor error:

```
Output importance file (     149) is not the expected length (149).
```

So, formatting of the word count size string.

Reply:
I think `status_code` was not being created because you previously `cd` into an unexpected directory before running the test. So it is created in some other directory.